src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_base.erl

```diff
@@ -839,6 +839,10 @@ malformed_request(Req, #base_state{resource_mod=Mod,
             lager:info("json too large (~p)", [Msg]),
             Req3 = wrq:set_resp_body(chef_json:encode({[{<<"error">>, Msg}]}), Req),
             {{halt, 413}, Req3, State1#base_state{log_msg = too_big}};
+        throw:{acl_constraint_violation, Violation} ->
+            Msg = chef_wm_malformed:malformed_request_message(Violation, Req, State),
+            Req3 = wrq:set_resp_body(chef_json:encode(Msg), Req),
+            {{halt, 403}, Req3, State1#base_state{log_msg = Msg}};
```

Comment:
I thought about 400 vs. 403. It comes down to how you want to parse things - you can say that it's malformed via our rules, or you can do as I did: it's a legal request that was sent across the wire just fine, but we're going to refuse to honor it, hence 403 forbidden (403 has a wider range than just you aren't allowed to access something, per my reading of the definition: http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html).

I'm not wedded to 403 and could easily go with 400. Anyone else have opinions for consideration?